src/explore-education-statistics-common/src/services/permalinkSnapshotService.ts

```diff
@@ -40,25 +40,28 @@ export interface PermalinkSnapshot {
 }
 
 interface CreatePermalink {
-  query: TableDataQuery;
+  query: ReleaseTableDataQuery;
   configuration: {
     tableHeaders: UnmappedTableHeadersConfig;
   };
 }
 
 const permalinkSnapshotService = {
```

Comment:
We can just call this `permalinkService` again too

Reply:
Renamed now, thanks. I didn't go as far as renaming `PermalinkSnapshot` to `Permalink`. Would you also recommend making that change?